apps/test/unit/code-studio/initApp/projectTest.js

```diff
@@ -975,6 +976,32 @@ describe('project.js', () => {
       });
     });
   });
+
+  describe('project.isCodeDifferent', () => {
```

Comment:
Do these cases cover the cases when there are no starting blocks?  If not, that might be a useful case to add?

Reply:
Good call! I'll add that in.